src/androidTest/java/androidx/view/TextViewTest.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
```

Comment:
Oops sorry I missed this, the copyright year should be 2018

Reply:
Sure, I copied it from other classes . I can correct this in my PR and all other in another or in the same?

Reply:
Existing files that say 2017 are correct, you should only worry about new files.